include/otf2xx/definition/group.hpp

```diff
@@ -100,6 +100,18 @@ namespace definition
             return this->data_->name();
         }
 
+        /**
+         * \brief sets the name of the group definition
+         *
+         * \param a string definition containing the new name
+         *
+         */
+        void name(const otf2::definition::string& name) const
```

Comment:
I'd rather not have a `const` setter.

Reply:
This does actually work because the compiler only enforces bitwise constant for **the current object**. The setter is actually const for the compiler because it neither sets a value in the current object nor calls a non-const method on `this`. However, it manipulates the implementation object, which means it is not **logically** `const`.